app/models/manageiq/providers/storage_manager.rb

```diff
@@ -12,5 +12,13 @@ class StorageManager < ManageIQ::Providers::BaseManager
                :foreign_key => :parent_ems_id,
                :class_name  => "ManageIQ::Providers::BaseManager",
                :autosave    => true
+
+    def cinder_service_available?
+      raise NotImplementedError, _("cinder_service_available? must be implemented in a subclass")
+    end
+
+    def swift_service_available?
+      raise NotImplementedError, _("swift_service_available? must be implemented in a subclass")
+    end
```

Comment:
Does UI need to access StorageManager to know if these services are available? If not, I can remove them.